packages/babel-plugin-proposal-private-methods/test/fixtures/accessors-loose/options.json

```diff
@@ -0,0 +1,15 @@
+{
+  "plugins": [
+    [
+      "external-helpers",
+      {
+        "helperVersion": "7.1.6"
```

Comment:
Could you use something like `7.1000.0`? So that we don't have to change this file if the helpers change in a futue version.

Reply:
Sure, done.